runtime/wasm/src/host_exports.rs

```diff
@@ -97,6 +99,20 @@ impl<C: Blockchain> HostExports<C> {
         }
     }
 
+    fn check_entity_type_access(&self, entity_type: &EntityType) -> Result<(), HostExportError> {
```

Comment:
even though this is not exported I think we should a bit of context here as a comment

Reply:
I've written a short comment and linked it to the one on `enum EntityTypeAccess`.